.gitignore

```diff
@@ -1,6 +1,7 @@
 /target
 Cargo.lock
 /trace.json
+.DS_Store
```

Comment:
On principle, I'm opposed to this - it should be configured in your global gitignore, as it's globally applicable to your system

But practically, I think we should probably add this change. There's no cost to having something stupid in the gitignore.

Reply:
I'll remove it

Reply:
I would keep it, although I agree that this should probably be ignored globally, I'm not sure how many really have it configured (and then it's annoying, either when it's committed, or when it's always there as unstaged file)